### PR TITLE
Add documentation on normalization for distillation

### PR DESCRIPTION
**Pull request by liopeer**
## What has changed and why?
- "default" teachers require imagenet normalization, however this was not properly documented
- this could cause issues for people using distillation, but changing the normalization, since it changes it for both, student & teacher, which share the augmentation pipeline
- this PR adds documentation for the usual DINOv2/DINOv3 and RADIO-based backbones and issues a warning is someone alters normalization params for these backbones
- also fixes an issue in the CHANGELOG introduced by #701 

## How has it been tested?
- unit tests

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [x] Yes
- [ ] Not needed (internal change)

## Did you update the documentation?

- [ ] Yes
- [x] Not needed (internal change without effects for user)

### CODE REVIEW

File: docs/source/settings/pretrain_settings.md [hunk 1]
```diff
@@ -571,11 +571,21 @@ lightly_train.pretrain(
...
+
 #### `normalize`
 
 Dictionary specifying the mean and standard deviation used for input normalization.
-ImageNet statistics are used by default. Change these values when working with datasets
-that have different color distributions.
+ImageNet statistics are used by default. For non-distillation methods, you can change
+these values if your model or training setup expects different input statistics.
+
+```{important}
```
Comment (mrpositron) [inline docs/source/settings/pretrain_settings.md]: I'll make it a {warning}. Wdyt?

File: docs/source/pretrain_distill/methods/distillation.md [hunk 2]
```diff
@@ -76,6 +76,22 @@ lightly-train pretrain out=out/my_experiment data=my_data_dir model="torchvision
...
+Distillation passes the same transformed image to the student and teacher. LightlyTrain
+uses ImageNet normalization by default, with mean `(0.485, 0.456, 0.406)` and standard
+deviation `(0.229, 0.224, 0.225)`. Keep this default when using LightlyTrain's
+`dinov2/*`, `dinov3/*`, or `radio/*` models as the teacher, or else the features might
+degrade. This also applies to TIPSv2, EUPE, and LingBot variants in those packages.
+
+Custom teachers can have a different input contract. In that case, set
+`transform_args.normalize` to the statistics expected by the custom teacher. If your
+downstream student requires a different normalization from the teacher, you can either
+denormalize in the students forward pass or just adjust the normalization during the
```
Comment (mrpositron) [inline docs/source/pretrain_distill/methods/distillation.md]: should be "student's"

**Review by mrpositron — APPROVED** [review]
LGTM 🚀

Left some comments

File: docs/source/settings/pretrain_settings.md (hunk 1 above)
Comment (liopeer) [reply docs/source/settings/pretrain_settings.md]: Done!

File: docs/source/pretrain_distill/methods/distillation.md (hunk 2 above)
Comment (liopeer) [reply docs/source/pretrain_distill/methods/distillation.md]: Done!
